system-variables.md

```diff
@@ -472,7 +472,8 @@ MPP 是 TiFlash 引擎提供的分布式计算框架，允许节点之间的数
 
 > **注意：**
 >
-> 目前该特性可能造成事务语义的变化，且与 TiDB Binlog 存在部分不兼容的场景，可以参考[事务语义行为区别](https://github.com/pingcap/tidb/issues/21069)和[与 TiDB Binlog 兼容问题汇总](https://github.com/pingcap/tidb/issues/20996)了解更多关于该特性的使用注意事项。
+> - 目前该特性可能造成事务语义的变化，且与 TiDB Binlog 存在部分不兼容的场景，可以参考[事务语义行为区别](https://github.com/pingcap/tidb/issues/21069)和[与 TiDB Binlog 兼容问题汇总](https://github.com/pingcap/tidb/issues/20996)了解更多关于该特性的使用注意事项。
+> - 从 v5.1.0 版本起，TiDB 开始支持 Reorg 类型变更的 [MODIFY/CHANGE COLUMN](/sql-statements/sql-statement-modify-column.md)，但不兼容此变量功能。
```

Comment:
I think we don't need to change this because it is mentioned in the previous paragraph:

```
- `MODIFY COLUMN` 或 `CHANGE COLUMN` 类型的 DDL 操作，且只对增大字段长度的操作生效。
```

Reply:
normal `MODIFY COLUMN` is compatible with `tidb_enable_amend_pessimistic_txn` except reorg type